MemberExpectations.md

```diff
@@ -0,0 +1,58 @@
+It is important that members of our leadership groups (TSC, CommComm) act
+in a way that not only complies with the code of conduct but that supports
```

Comment:
s/code of conduct/Code of Conduct, and make this a link to the doc

Reply:
done